.github/workflows/build-envoy-images-release.yaml

```diff
@@ -193,7 +195,7 @@ jobs:
           push: true
           tags: |
             quay.io/${{ github.repository_owner }}/cilium-envoy:latest
-            quay.io/${{ github.repository_owner }}/cilium-envoy:${{ github.sha }}
+            quay.io/${{ github.repository_owner }}/cilium-envoy:${{ env.ENVOY_MINOR_RELEASE }}-${{ github.sha }}
```

Comment:
This change is also in PR #147, but there you also remove the "latest" tag... Would it be possible to keep the `latest` tag for the latest `master` build, and not overwrite `latest` tag in any non-master builds?